### PR TITLE
Tweaks pursuant to AAAI15-era evaluation

### PR DESCRIPTION
**Pull request by bkisiel**
…rca Dec. 2015
- Addition to compute_metrics.py to output a table of the format Tom uses for graphs in our past AAAI papers.

### CODE REVIEW

File: scripts/import_predictions.py [hunk 1]
```diff
@@ -19,8 +19,9 @@
 from annotate.models import Task
 
 
+@transaction.atomic
 def main(data_file, method_name):
-    transaction.set_autocommit(False)
+    #transaction.set_autocommit(False)
```
Comment (matt-gardner) [inline scripts/import_predictions.py]: Delete the commented out line.

File: scripts/import_predictions.py [hunk 2]
```diff
@@ -40,8 +41,8 @@ def main(data_file, method_name):
                                 ranking=rank)
         prediction.save()
     sample_instances_for_pool(task_instances)
-    transaction.commit()
-    transaction.set_autocommit(True)
+    #transaction.commit()
+    #transaction.set_autocommit(True)
```
Comment (matt-gardner) [inline scripts/import_predictions.py]: Delete these too.

**Comment by matt-gardner** [comment]
Thanks!  If you could just remove the commented out lines, I'll merge this.

**Comment by bkisiel** [comment]
No problem -- good to finally tie up a loose end.  I pushed it to my 
branch.  Let me know if I need to make a new pull request.

**Pull request merged**